.circleci/config.yml

```diff
@@ -30,7 +30,7 @@ jobs:
 
       - run:
           name: Install tox
-          command: pip install tox
+          command: sudo pip install tox
```

Comment:
`pip install tox --user` won't do it?

Reply:
@swilly22 I tried it, 4 ways: 
* pip install tox - not working (not permission to install)
* pip install --user tox - not working (tox command not found on next command)
* sudo pip install --user tox - not working tox command not found
* sudo pip install tox - is only working option

Reply:
I think it should be `pip install tox --user`